[dtensor] delete lagging op db and update op db tests

wanchaol opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #92373
* __->__ #92290

We are now in pytorch core so don't need lagging op db anymore

wanchaol committed:
[dtensor] delete lagging op db and update op db tests

We are now in pytorch core so don't need lagging op db anymore

[ghstack-poisoned]

XilunWu reviewed (approved):
thanks for updating dtensor ops db test! The red flag should be gone after a rerun.

wanchaol committed:
Update on "[dtensor] delete lagging op db and update op db tests"


We are now in pytorch core so don't need lagging op db anymore

[ghstack-poisoned]
